We never really said this before (#58)
ietf-wg-ohai/draft-ohai-chunked-ohttp: draft-ietf-ohai-chunked-ohttp.md

If a request uses the media type "message/ohttp-chunked-req",
a successful corresponding response MUST use the media type "message/ohttp-chunked-res".

> Chunked Oblivious HTTP requests MUST contain Binary HTTP requests

Reviewer: ```suggestion
Chunked Oblivious HTTP requests are defined to contain Binary HTTP requests
```

I'm concerned that this text sounds like it is precluding using other media types, which is something that I believe needs to still be supported -- my implementation has tests to validate functionality for alternate media types. https://www.rfc-editor.org/rfc/rfc9458#section-4.6

in either Known- or Indeterminate-Length form;

> responses MUST contain Binary HTTP responses; see {{!BHTTP}}.

Reviewer 2: I tend to err on the side of stating the obvious, but I don't feel strongly

> responses MUST contain Binary HTTP responses; see {{!BHTTP}}.

Reviewer: I prefer brevity in this case.

If a request uses the media type "message/ohttp-chunked-req",
a successful corresponding response MUST use the media type "message/ohttp-chunked-res".

Chunked Oblivious HTTP requests are defined to contain Binary HTTP requests
in either Known- or Indeterminate-Length form;
responses contain Binary HTTP responses; see {{!BHTTP}}. As with non-chunked Oblivious HTTP, the encapsulation format

> of chunked Oblivious HTTP can be repurposed for other media types. See {{Section 4.6 of OHTTP}}.

Author: ```suggestion
of chunked Oblivious HTTP can be repurposed for other media types;
see {{Section 4.6 of OHTTP}}.
```

If a request uses the media type "message/ohttp-chunked-req",
a successful corresponding response MUST use the media type "message/ohttp-chunked-res".

Chunked Oblivious HTTP requests are defined to contain Binary HTTP requests
in either Known- or Indeterminate-Length form;
responses contain Binary HTTP responses; see {{!BHTTP}}. As with non-chunked Oblivious HTTP, the encapsulation format
of chunked Oblivious HTTP can be repurposed for other media types;

> see {{Section 4.6 of OHTTP}}.

Author: ```suggestion
responses contain Binary HTTP responses; see {{!BHTTP}}.
As with non-chunked Oblivious HTTP, 
the encapsulation format of chunked Oblivious HTTP can be repurposed
for other media types; see {{Section 4.6 of OHTTP}}.
```
